src/orquestra/sdk/_base/_config.py

```diff
@@ -214,9 +215,39 @@ def _resolve_auto_config(config_name) -> RuntimeConfiguration:
         "uri": uri,
         "token": passport_token,
     }
+
     return runtime_config
 
 
+def _resolve_local_auto_config(config_env: str) -> RuntimeConfiguration:
+    # if someone sets "auto" as CURRENT_CONFIG_ENV variable, we would get into infinite
+    # recursion here.
+    if config_env == AUTO_CONFIG_NAME:
+        raise exceptions.RuntimeConfigError(
+            f"{AUTO_CONFIG_NAME} can not be the value "
+            f"of {CURRENT_CONFIG_ENV} env variable."
+        )
+    try:
+        config = read_config(config_env)
+    except (exceptions.ConfigFileNotFoundError, exceptions.ConfigNameNotFoundError):
+        raise exceptions.RuntimeConfigError(
+            f"Couldn't find the config {config_env} specified in "
+            f"{CURRENT_CONFIG_ENV} env variable."
+        )
+    return config
+
+
+def _resolve_auto_config(config_name) -> RuntimeConfiguration:
+    # On studio we short-circuit to use internal URIs
+    if is_passport_file_available():
+        return _resolve_remote_auto_config(config_name)
+
+    if CURRENT_CONFIG_ENV in os.environ:
+        return _resolve_local_auto_config(os.environ[CURRENT_CONFIG_ENV])
+
+    return LOCAL_RUNTIME_CONFIGURATION
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** I think we should discuss this fallback. I think we should break previous behaviour and have "auto" fail when `ORQ_CURRENT_CONFIG` is not set.
This is because it is likely that someone will forget to set the environment variable and be confused why their workflow isn't running on cluster X. However, it's running locally.